tests/common/utilities.py

```diff
@@ -510,7 +513,10 @@ def compare_crm_facts(left, right):
     for k, v in list(left_acl_group.items()):
         lv = v
         rv = right_acl_group[k]
-        if lv['available'] != rv['available'] or lv['used'] != rv['used']:
+
+        # For Cisco-8000 devices, hardware counters are statistical-based with +/- 1 entry tolerance.
```

Comment:
same as above comment, this check needs to be added to take into effect only while running against cisco specific devices